### PR TITLE
Add 2023 Maine Tax Rules

### LINKED ISSUE #4182: Add 2023 Maine Tax Rules
2023 Income Tax Form Instructions: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/23_1040me_book_gen_instr.pdf

### PR DESCRIPTION
**Pull request by jialyu0411**
Fixes #4182

### CODE REVIEW

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml [hunk 1]
```diff
@@ -3,42 +3,54 @@ SINGLE:
...
   1:
     2021-01-01: 3_400
     2022-01-01: 3_450
+    2023-01-01: 3_700
   2:
     2021-01-01: 3_400
     2022-01-01: 3_450
-SURVIVING_SPOUSE:
+    2023-01-01: 3_700
+WIDOW:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:41]:
```suggestion
SURVIVING_SPOUSE:
```

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/amount.yaml [hunk 2]
```diff
@@ -20,5 +22,7 @@ HEAD_OF_HOUSEHOLD:
   2021-01-01: 15
 JOINT:
   2021-01-01: 20
-SURVIVING_SPOUSE:
+  2023-01-01: 15
+WIDOW:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/amount.yaml:26]:
```suggestion
SURVIVING_SPOUSE:
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:1]: Is this section of the [legal code](https://legislature.maine.gov/statutes/36/title36sec5219-KK.html) mirrored in the tax form and our current code:
image

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/phase_out/start.yaml [hunk 3]
```diff
@@ -3,6 +3,8 @@ metadata:
   unit: currency-USD
   label: Maine standard/itemized exemption phase-out start
   reference:
+    - title: Maine form 1040 Schedule 2 (2023)
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/deductions/phase_out/start.yaml:6]: could we find the legal code here?

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]
A couple of questions:
Do any of the amounts just adjust every year with inflation? 

Could you summarize the changes in a comment in the issue:
https://github.com/PolicyEngine/policyengine-us/issues/4182

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml [hunk 4]
```diff
@@ -56,6 +68,8 @@ metadata:
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/21_1040me_sched_pstfc_dwnldff.pdf#page=2
     - title: 2022 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:69]: Is this uprated with inflation?

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:69]: same question for all parameter files which seem to be updated every year

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: new credit:?
image

[Page 2](https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/23_1040me_book_gen_instr.pdf#page=2)

Comment (PavelMakarchuk) [reply changelog_entry.yaml:1]: also change in pension income deduction
image

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/me/tax/income/agi/subtractions/pension_exclusion/cap.yaml [hunk 5]
```diff
@@ -7,9 +7,13 @@ values:
 metadata:
   unit: currency-USD
   period: year
+  uprating: gov.irs.uprating
+  #uprating: gov.states.me.cpi
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/agi/subtractions/pension_exclusion/cap.yaml:11]:
```suggestion
```

File: policyengine_us/parameters/gov/states/me/tax/income/credits/eitc/rate/no_qualifying_child.yaml [hunk 6]
```diff
@@ -5,9 +5,11 @@ values:
   2022-01-01: 0.50
 metadata:
   unit: /1
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/eitc/rate/no_qualifying_child.yaml:7]:
```suggestion
  unit: /1
  period: year
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/eitc/rate/no_qualifying_child.yaml:7]: Add period everywhere when its missing while we are in here

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml [hunk 7]
```diff
@@ -3,42 +3,54 @@ SINGLE:
...
     2022-01-01: 2_800
+    2023-01-01: 3_000
   1:
     2021-01-01: 3_400
     2022-01-01: 3_450
+    2023-01-01: 3_700
   2:
     2021-01-01: 3_400
     2022-01-01: 3_450
+    2023-01-01: 3_700
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:40]: is this uprated and then rounded?

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml [hunk 8]
```diff
@@ -56,7 +68,11 @@ metadata:
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/21_1040me_sched_pstfc_dwnldff.pdf#page=1
     - title: 2022 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit 
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_pstfc_ff.pdf#page=1
+    - title: 2023 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit
+      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/23_1040me_sched_pstfc_ff.pdf#page=3
   period: year
+  uprating: gov.irs.uprating
+  #uprating: gov.states.me.cpi
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml:75]:
```suggestion
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml:75]: delete comments everywhere

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml [hunk 9]
```diff
@@ -56,7 +68,11 @@ metadata:
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/21_1040me_sched_pstfc_dwnldff.pdf#page=1
     - title: 2022 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit 
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_pstfc_ff.pdf#page=1
+    - title: 2023 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit
+      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/23_1040me_sched_pstfc_ff.pdf#page=3
   period: year
+  uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml:74]: this looks rounded to me

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/veterans.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/veterans.yaml:1]: rename to `veterants_in_effect.yaml`

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/start.yaml [hunk 10]
```diff
@@ -1,6 +1,9 @@
 description: Maine reduces the sales tax fairness credit for filers with income above this amount, based on filing status.
 metadata:
   unit: currency-USD
+  period: year
+  uprating: gov.irs.uprating
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/start.yaml:5]: rounded?

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py [hunk 11]
```diff
@@ -47,4 +47,18 @@ def formula(tax_unit, period, parameters):
 
         cap = max_(head_cap, spouse_cap)
 
-        return min_(uncapped_credit, cap)
+        credit = min_(uncapped_credit, cap)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py:50]: use an if statement here instead

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/width.yaml [hunk 12]
```diff
@@ -18,4 +21,4 @@ HEAD_OF_HOUSEHOLD:
 JOINT:
   2021-01-01: 125_000
 SURVIVING_SPOUSE:
-  2021-01-01: 62_500
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/width.yaml:21]: Was this change intended?

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml (hunk 7 above)
Comment (jialyu0411) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:40]: It is uprated but not rounded.

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml (hunk 9 above)
Comment (jialyu0411) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml:74]: It is uprated but not rounded.

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/start.yaml (hunk 10 above)
Comment (jialyu0411) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/start.yaml:5]: It is uprated but not rounded.

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/width.yaml (hunk 12 above)
Comment (jialyu0411) [reply policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/width.yaml:21]: The numerator of the fraction is the taxpayer's Maine adjusted gross income less the applicable amount, except that the numerator may not be less than zero, and the denominator is $62,500 in the case of a married individual filing a separate return and $125,000 in all other cases.[§5126-A](https://legislature.maine.gov/statutes/36/title36sec5126-A.html)

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml [hunk 13]
```diff
@@ -1,21 +1,43 @@
...
+      href: https://legislature.maine.gov/statutes/36/title36sec5403.html
+    - title: 2022 Instructions for Form 1040ME Line 18 Exemption
+      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_book_gen_instruc_revisedFeb23.pdf#page=4
+    - title: 2021 Instructions for Form 1040ME
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/21_1040me_book_gen_instr_revMay2022.pdf#page=4
     - title: §5126-A. Personal exemptions on or after January 1, 2018
       href: https://legislature.maine.gov/statutes/36/title36sec5126-A.html
   breakdown:
     - filing_status
 SINGLE:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml:24]: We need to add the uprating to each component of the breakdown file:

See example
image

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml [hunk 14]
```diff
@@ -1,21 +1,43 @@
 description: Maine starts phasing out its personal exemption for filers with Maine AGI above this threshold.
 metadata:
   unit: currency-USD
+  period: year
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: downwards
+      interval: 50
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml:9]: remove here

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml (hunk 13 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml:24]: all breakdown files

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py [hunk 15]
```diff
@@ -47,4 +47,18 @@ def formula(tax_unit, period, parameters):
...
+        # compute permanently and totally disabled veterans's credit
+        disabled_veterans = add(
+            tax_unit, period, ["is_fully_disabled_service_connected_veteran"]
+        )
+        permanently_disabled_veterans = add(
+            tax_unit, period, ["is_permanently_disabled_veteran"]
+        )
+        qualified_veterans = (
+            disabled_veterans and permanently_disabled_veterans
+        )
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py:61]:
```suggestion
        person = tax_unit.members
        disabled_service_connected_veteran = person("is_fully_disabled_service_connected_veteran", period)
        permanently_disabled_veteran = person("is_permanently_disabled_veteran", period)
        qualified_disabled_veteran_present = tax_unti.any(disabled_service_connected_veteran | permanently_disabled_veteran)
```

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py (hunk 11 above)
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py:50]:
```suggestion
        base_credit = min_(uncapped_credit, cap)
```

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py [hunk 16]
```diff
@@ -47,4 +47,18 @@ def formula(tax_unit, period, parameters):
...
+        qualified_veterans_credit = credit * 2
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py:62]: parameterize the number "2"

File: policyengine_us/tests/policy/baseline/gov/states/me/tax/income/credits/property_tax_fairness_credit/me_property_tax_fairness_credit.yaml [hunk 17]
```diff
@@ -109,3 +109,39 @@
...
+    is_fully_disabled_service_connected_veteran: false
+    is_permanently_disabled_veteran: true
+    ctc_qualifying_children: 5
+    utilities_included_in_rent: false
+    utility_expense: 0
+    me_sales_and_property_tax_fairness_credit_income: 0
+    age_head: 64
+    age_spouse: 64
+  output:
+    me_property_tax_fairness_credit: 1_000
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/me/tax/income/credits/property_tax_fairness_credit/me_property_tax_fairness_credit.yaml:147]: need to create tests with multiple people in the household to show how it behaves

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py [hunk 18]
```diff
@@ -47,4 +47,18 @@ def formula(tax_unit, period, parameters):
 
         cap = max_(head_cap, spouse_cap)
 
-        return min_(uncapped_credit, cap)
+        credit = min_(uncapped_credit, cap)
+
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py:51]: create an if statement with the veterans_in_effect.yaml parameter

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py (hunk 15 above)
Comment (PavelMakarchuk) [reply policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py:61]: need to check whether this only applied to head and spouse or also dependents

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py [hunk 19]
```diff
@@ -47,4 +47,18 @@ def formula(tax_unit, period, parameters):
...
+        )
+        permanently_disabled_veterans = add(
+            tax_unit, period, ["is_permanently_disabled_veteran"]
+        )
+        qualified_veterans = (
+            disabled_veterans and permanently_disabled_veterans
+        )
+        qualified_veterans_credit = credit * 2
+
+        return qualified_veterans_credit if qualified_veterans else credit
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py:64]: where statement instead

File: policyengine_us/tests/policy/baseline/gov/states/me/tax/income/credits/property_tax_fairness_credit/me_property_tax_fairness_credit.yaml (hunk 17 above)
Comment (PavelMakarchuk) [reply policyengine_us/tests/policy/baseline/gov/states/me/tax/income/credits/property_tax_fairness_credit/me_property_tax_fairness_credit.yaml:147]: 2023 test as well as 2022 test to show when the veteran condition

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/veterants_in_effect.yaml [hunk 20]
```diff
@@ -0,0 +1,13 @@
+description: Maine provides additional credit for permanently and totally disabled veterans if this is true.
+metadata:
+  unit: bool
+  period: year
+  label: Maine provides additional credit for permanently and totally disabled veterans
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/veterants_in_effect.yaml:5]:
```suggestion
  label: Maine additional credit for permanently and totally disabled veterans in effect
```

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/veterants_in_effect.yaml [hunk 21]
```diff
@@ -0,0 +1,13 @@
+description: Maine provides additional credit for permanently and totally disabled veterans if this is true.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/veterants_in_effect.yaml:1]:
```suggestion
description: Maine provides additional credit for permanently and totally disabled veterans if this is true.
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml (hunk 13 above)
Comment (jialyu0411) [reply policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml:24]: > all breakdown files

Not sure how to add values for 'amount' in 'property_tax'.
15b9f1c006c47014c307b34c0872a19

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py (hunk 15 above)
Comment (jialyu0411) [reply policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py]: According to the order of items listed on Form 1040ME, especially from items 12 to 14, I think `permanently and totally disabled veterans' credit` was calculated after the `base_credit` was calculated. And this part is applying upon the previously calculated total base credits which had already taken the filling status and the number of dependents into account.

Comment (PavelMakarchuk) [reply policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py]: Yes that is correct, but the question remains, if the TaxUnit includes a senior who is a disabled veteran, who is not the main head or spouse of the household (e.g. father of the head/spouse) is this person considered for the credit or is limited to the head and spouse themselves

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml [hunk 22]
```diff
@@ -52,11 +64,20 @@ metadata:
...
+    - title: §5403. Annual adjustments for inflation 6-Property tax fairness credit
+      href: https://legislature.maine.gov/statutes/36/title36sec5403.html
     - title: 2021 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit 
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/21_1040me_sched_pstfc_dwnldff.pdf#page=2
     - title: 2022 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit 
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_pstfc_ff.pdf#page=2
+    - title: 2023 Form 1040ME Property Tax Fairness Credit / Sales Tax Fairness Credit Line 7
+      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/23_1040me_sched_pstfc_ff.pdf#page=2 
   period: year
+  uprating:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:76]: this uprating needs to be applied to each breakdown

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/amount.yaml:76]: all breakdown yaml files

File: policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py (hunk 15 above)
Comment (jialyu0411) [reply policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py]: A taxpayer can claim his/her parent(s) as dependents when he/she files under married jointly or as HOH, but the number of dependents does not determine whether the tax-filling entity will receive the extra credit or not.
Given your example where a senior disabled vet who is neither under Single, HOH, Married Jointly nor Qualified Surviving Spouse status, then he/she must be claimed as a dependent whose disability and military status are not considered when determining the eligibility for the VA credits on item 13, Form 1040ME. So for this filling entity, the extra VA credit won't be applied. However, if the senior disabled vet is filling on his/her own behalf and this would make him/her file under Single, HOH, Qualified Surviving Spouse or Married Jointly thus making the VA credit available.

File: policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/start.yaml [hunk 23]
```diff
@@ -1,28 +1,72 @@
...
+  values:
+    2021-01-01: 21_350
+    2022-01-01: 21_850
+    2023-01-01: 23_300
+  metadata:
+    uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: downwards
+      interval: 50
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/start.yaml:30]:
```suggestion
      parameter: gov.irs.uprating
      rounding:
        type: downwards
        interval: 50
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/sales_tax/reduction/start.yaml:30]: nest the parameter and rounding functions

**Comment by hua7450** [comment]
PR #4672 supersedes this PR.
